javascript/javascript3/week1/weather.js

```diff
@@ -0,0 +1,56 @@
+//Weather app
+document.addEventListener("DOMContentLoaded", fetchweather);
+
+function fetchweather() {
+  const weatherButton = document.getElementById("btn");
+  weatherButton.addEventListener("click", (ev) => {
+    ev.preventDefault(); // to stop the page reload
+
+    let inputCity = document.getElementById("city-input");
+    fetch(
+      `https://api.openweathermap.org/data/2.5/weather?q=${inputCity.value.trim()}&units=metric&appid=750dcc6f8327e6cb1989916cbe0c8c7b`
+    )
+      .then((response) => response.json())
+      .then((weather) => {
+        if (inputCity.value == "" || !inputCity.value.length) {
+          const errorp = document.getElementById("error-p");
+          errorp.innerText = "error type a city name first";
```

Comment:
To show a message to the user, It would be better not to write the word "error". We use error messages mainly inside the program. Also, please consider Using capital letters for the first of your message to the user.